project/CrossJava.scala

```diff
@@ -30,10 +29,20 @@ case class JavaVersion(numbers: Vector[Long], vendor: Option[String]) {
   }
 }
 object JavaVersion {
+
+  val specificationVersion: String = sys.props("java.specification.version")
+  val version: String = sys.props("java.version")
+  val is1x: Boolean = version.startsWith("1.")
```

Comment:
the version numbering is pretty confusing, sometimes using 1.8 sometimes 8, sometimes 11, don't know if there is also 1.9 and 1.11? What is it that we are interested in here and can we give it a better name than `is1x`? E.g. `isJdk8`?

Reply:
Also, why are we looking at `specification.version` for some things and `java.version` for some? @raboof do you know?

Reply:
Hmm, probably because `java.version` is more well-known but `java.specification.version` is actually more convenient here? I think it'd make sense to move towards using `java.specification.version` everywhere (unless you *really* need the specific version).

I agree `isJdk8` would be better than `is1x`, hides some of the confusing version numbering from the JDK.

Reply:
I like isJdk8`isJdk8` much better, thanks! Just couldn't think of anything. Changing.

Reply:
Where relevant, shall I migrate java.version usages to java.specification.version here or in a follow up PR?